SAMtemplates/state_machines/main.yaml

```diff
@@ -30,7 +30,7 @@ Parameters:
   EnableSplunk:
     Type: String
 
-  FihrValidatorStackName:
+  FhirValidatorStackName:
```

Comment:
Awestruck that you spotted this

Reply:
I would take the credit but I only saw it when cloudformation kicked off when I tried to import it spelt correctly 😹